lib/syskit/dynamic_port_binding.rb

```diff
@@ -203,6 +203,10 @@ def initialize(
             # Method called by {Accessor} to create the accessor object from a
             # port
             def create_accessor(port)
+                if port.respond_to?(:init_policy) &&
+                   [true, false].include?(port.init_policy)
+                    policy.merge(init: port.init_policy)
```

Comment:
This overrides an explicit `init` flag that may be given in `policy`. The explicit value (from `policy` should take precedence